.github/workflows/release.yml

```
@@ -0,0 +1,24 @@
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20.x'
+      - run: npm install
+      - run: npm run check
+      - run: npm run test
+
+      # Use node pointing to the internal releasing registry.
```

```suggestion
      # Use node pointing to the Google releasing registry.
```

Remember that this code _is_ open-source, so it's worth documenting in a way that makes sense to non-Google readers.